README.md

```diff
@@ -534,6 +534,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Hamcrest](http://hamcrest.org/JavaHamcrest/) - Matchers that can be combined to create flexible expressions of intent.
 * [JMockit](http://jmockit.org/) - Mocks static, final methods and more.
 * [JUnit](http://junit.org/) - Common testing framework.
+* [JUnitParams](https://pragmatists.github.io/JUnitParams/) - Creation of readable and maintainable parametrised tests. It provides much better functionalities than JUnit Parametrised runner.
```

Comment:
I'm sorry to be pedantic but if you make a claim "to be much better than x" you have to back it up with arguments. I would remove the second sentence.